modules/tracing/mule-tracer-customization-impl/src/main/java/org/mule/runtime/tracer/customization/impl/info/ExecutionInitialSpanInfo.java

```diff
@@ -64,7 +63,8 @@ public ExecutionInitialSpanInfo(Component component, String apiId, InitialExport
       name = overriddenName;
       initialExportInfo = initialExportInfoProvider.getInitialExportInfo(name);
     }
-    this.isPolicySpan = isComponentOfName(component, EXECUTE_NEXT) || component instanceof PolicyChain;
+    this.isPolicySpan = isComponentOfName(component, EXECUTE_NEXT) || component instanceof PolicyChain
```

Comment:
Change name to reflect that this no longer includes only policies

Reply:
We have a specific task for this, scheduled for the next sprint: W-13313594

Reply:
I think the TODO was not added.